### PR TITLE
feat: optimize UserEditPage & PermissionLIstPage display

### PR DESCRIPTION
**Pull request by SealinGp**
- fix:PermissionList may broken if text undefined
- feat: update applications when organizations changed

Signed-off-by: SealinGp <sealingp@163.com>

### CODE REVIEW

**Comment by SealinGp** [comment]
> @SealinGp use the same committer name as github
> 
> image

done

File: web/src/locales/de/data.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "Sync policies successfully": "Sync policies successfully"
   },
   "application": {
+    "Empty": "Empty",
```
Comment (hsluoyz) [inline web/src/locales/de/data.json:19]: This is wrong, update i18n as: https://casdoor.org/docs/contributing#how-to-update-i18n-data

File: web/src/UserEditPage.js [hunk 2]
```diff
@@ -115,7 +115,17 @@ class UserEditPage extends React.Component {
     return value;
   }
 
+  getEmptyText(text) {
```
Comment (hsluoyz) [inline web/src/UserEditPage.js:118]: provide a screenshot of the UI effect

**Review by hsluoyz — CHANGES_REQUESTED** [review]
Don't update `yarn.lock`

File: web/src/locales/de/data.json (hunk 1 above)
Comment (SealinGp) [reply web/src/locales/de/data.json]: done

File: web/src/UserEditPage.js (hunk 2 above)
Comment (SealinGp) [reply web/src/UserEditPage.js:118]: image
image

File: web/src/locales/de/data.json [hunk 3]
```diff
@@ -173,6 +173,7 @@
     "Edit": "Bearbeiten",
```
Comment (hsluoyz) [inline web/src/locales/de/data.json:173]: Don't update `yarn.lock`

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: web/src/UserEditPage.js (hunk 2 above)
Comment (hsluoyz) [reply web/src/UserEditPage.js:118]: Don't do this. This code is fragile, and maybe don't work for other languages.

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**